package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simple-icons",
-  "version": "1.9.13",
+  "version": "1.9.19",
```

Comment:
I think maintainers will take care for version update upon release.
Could you restore this value, and the one in lock file? We'll end up with one new file and one modified only then.

Reply:
In this regard no, I cannot remember an instance where a contributor changed the `"version"` in the package.json so we've never considered it. That being said, I think the Contributing Guidelines are clear on what you **should** change (in the case you're adding/updating an icon) and other than that contributors **shouldn't** change anything.

But don't worry about having changed it, that's what PR reviews are for 😃